src/main/java/com/virtuslab/using_directives/custom/CommentExtractor.java

```diff
@@ -50,12 +50,6 @@ private boolean skipComment() {
           while (reader.ch != CR && reader.ch != LF && reader.ch != SU) {
             reader.nextChar();
           }
-          while (reader.ch == CR) {
```

Comment:
Why is this change needed? I would assume, that CR and LF after comments should be ignored.

Reply:
With this piece of code, I had an issue that I copied line with CR but without LF (I think it's an issue that we need to keep some information to decide if we should move 1 char back or 2 chars back). Without skipping EOL the problem is gone and the only side-effect is that we've got one extra empty line at the top of file. (Which isn't a problem I think, since the trimmed file usually goes to compiler.

I think that we can stay with this solution and create some enhancement issue to not forget about that.

Reply:
Sure, makes sense